analyses/TX_cd_2020/doc_TX_cd_2020.md

```diff
@@ -0,0 +1,58 @@
+# 2020 Texas Congressional Districts
```

Comment:
these look great; I'm just going to tidy up a few things but nothing substantive